Cargo.toml

```diff
@@ -44,37 +44,34 @@ incremental = false
 alphanet-node = { path = "crates/node" }
 alphanet-precompile = { path = "crates/precompile" }
 
-alloy = { git = "https://github.com/alloy-rs/alloy", rev = "5796024", features = [
+alloy = { version = "0.2", features = [
     "contract",
     "providers",
     "provider-http",
     "signers",
 ] }
-alloy-network = { git = "https://github.com/alloy-rs/alloy", rev = "5796024" }
-alloy-signer-wallet = { git = "https://github.com/alloy-rs/alloy", rev = "5796024" }
+alloy-network = { version = "0.2" }
+alloy-signer-local = { version = "0.2", features = ["mnemonic"] }
 
 # tokio
 tokio = { version = "1.21", default-features = false }
 
 # reth
-reth = { git = "https://github.com/paradigmxyz/reth.git", rev = "40d8ac0", features = ["optimism"] }
-reth-node-api = { git = "https://github.com/paradigmxyz/reth.git", rev = "40d8ac0" }
-reth-node-core = { git = "https://github.com/paradigmxyz/reth.git", rev = "40d8ac0" }
-reth-node-optimism = { git = "https://github.com/paradigmxyz/reth.git", rev = "40d8ac0" }
-reth-primitives = { git = "https://github.com/paradigmxyz/reth.git", rev = "40d8ac0" }
-reth-tracing = { git = "https://github.com/paradigmxyz/reth.git", rev = "40d8ac0" }
+reth = { git = "https://github.com/paradigmxyz/reth.git", rev = "539b4e4", features = ["optimism"] }
+reth-chainspec = { git = "https://github.com/paradigmxyz/reth.git", rev = "539b4e4" }
+reth-cli-util = { git = "https://github.com/paradigmxyz/reth.git", rev = "539b4e4" }
+reth-node-api = { git = "https://github.com/paradigmxyz/reth.git", rev = "539b4e4" }
+reth-node-builder = { git = "https://github.com/paradigmxyz/reth.git", rev = "539b4e4" }
+reth-node-core = { git = "https://github.com/paradigmxyz/reth.git", rev = "539b4e4" }
+reth-node-optimism = { git = "https://github.com/paradigmxyz/reth.git", rev = "539b4e4" }
+reth-primitives = { git = "https://github.com/paradigmxyz/reth.git", rev = "539b4e4" }
+reth-tracing = { git = "https://github.com/paradigmxyz/reth.git", rev = "539b4e4" }
 
 # misc
 clap = "4"
 eyre = "0.6.12"
 tracing = "0.1.0"
-eip3074-instructions = { git = "https://github.com/paradigmxyz/eip3074-instructions.git", rev = "a9022e4" }
+eip3074-instructions = { git = "https://github.com/emostov/eip3074-instructions.git", branch = "bump-revm" }
```

Comment:
could you perhaps also bump the eip3074 instructions repo?

Reply:
pr here https://github.com/paradigmxyz/eip3074-instructions/pull/2

Reply:
actually looks like i need to bump msrv here and in eip3074 prs